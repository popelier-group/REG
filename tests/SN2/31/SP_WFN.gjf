# b3lyp/6-311++g(d,p) output=wfn

Single Point Energy

-1 1
 C                 -1.49748500   -0.00010300    0.00009000
 H                 -1.16687000   -0.87482400   -0.54520800
 H                 -1.16674600    0.90947000   -0.48476000
 H                 -1.16695400   -0.03501700    1.03029400
 Br                 1.75804800    0.00001700    0.00000000
 Cl                -3.35282500    0.00000700   -0.00004000

frame.wfn
