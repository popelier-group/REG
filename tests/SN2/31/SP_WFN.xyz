
6

C       -1.357531    0.018931    0.160587
H       -0.934127   -0.532296    0.990449
H       -1.142606   -0.468949   -0.781523
H       -1.010919    1.044423    0.156457
Br        1.875166   -0.026033   -0.221704
Cl       -3.199880    0.044376    0.378279
