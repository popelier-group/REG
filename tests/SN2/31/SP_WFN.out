 Entering Gaussian System, Link 0=/opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/g16
 Initial command:
 /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe "/scratch/v69787ff/gau_temp_1513317.31/Gau-38900.inp" -scrdir="/scratch/v69787ff/gau_temp_1513317.31/"
 Default is to use a total of   2 processors:
                                2 via shared-memory
                                1 via Linda
 Entering Link 1 = /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe PID=     38901.
  
 Copyright (c) 1988-2019, Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 16 program.  It is based on
 the Gaussian(R) 09 system (copyright 2009, Gaussian, Inc.),
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 16, Revision C.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, 
 G. A. Petersson, H. Nakatsuji, X. Li, M. Caricato, A. V. Marenich, 
 J. Bloino, B. G. Janesko, R. Gomperts, B. Mennucci, H. P. Hratchian, 
 J. V. Ortiz, A. F. Izmaylov, J. L. Sonnenberg, D. Williams-Young, 
 F. Ding, F. Lipparini, F. Egidi, J. Goings, B. Peng, A. Petrone, 
 T. Henderson, D. Ranasinghe, V. G. Zakrzewski, J. Gao, N. Rega, 
 G. Zheng, W. Liang, M. Hada, M. Ehara, K. Toyota, R. Fukuda, 
 J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, H. Nakai, 
 T. Vreven, K. Throssell, J. A. Montgomery, Jr., J. E. Peralta, 
 F. Ogliaro, M. J. Bearpark, J. J. Heyd, E. N. Brothers, K. N. Kudin, 
 V. N. Staroverov, T. A. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. P. Rendell, J. C. Burant, S. S. Iyengar, 
 J. Tomasi, M. Cossi, J. M. Millam, M. Klene, C. Adamo, R. Cammi, 
 J. W. Ochterski, R. L. Martin, K. Morokuma, O. Farkas, 
 J. B. Foresman, and D. J. Fox, Gaussian, Inc., Wallingford CT, 2019.
 
 ******************************************
 Gaussian 16:  ES64L-G16RevC.01  3-Jul-2019
                11-Nov-2020 
 ******************************************
 --------------------------------
 # b3lyp/6-311++g(d,p) output=wfn
 --------------------------------
 1/38=1,172=1/1;
 2/12=2,17=6,18=5,40=1/2;
 3/5=4,6=6,7=1111,11=2,25=1,30=1,74=-5/1,2,3;
 4//1;
 5/5=2,38=5/2;
 6/7=2,8=2,9=2,10=2,28=1/1;
 99/5=1,6=100,9=1/99;
 -------------------
 Single Point Energy
 -------------------
 Symbolic Z-matrix:
 Charge = -1 Multiplicity = 1
 C                    -1.49749  -0.0001    0.00009 
 H                    -1.16687  -0.87482  -0.54521 
 H                    -1.16675   0.90947  -0.48476 
 H                    -1.16695  -0.03502   1.03029 
 Br                    1.75805   0.00002   0. 
 Cl                   -3.35283   0.00001  -0.00004 
 
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -1.497485   -0.000103    0.000090
      2          1           0       -1.166870   -0.874824   -0.545208
      3          1           0       -1.166746    0.909470   -0.484760
      4          1           0       -1.166954   -0.035017    1.030294
      5         35           0        1.758048    0.000017    0.000000
      6         17           0       -3.352825    0.000007   -0.000040
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3          4          5
     1  C    0.000000
     2  H    1.082494   0.000000
     3  H    1.082493   1.785318   0.000000
     4  H    1.082492   1.785352   1.785342   0.000000
     5  Br   3.255533   3.101249   3.101051   3.101350   0.000000
     6  Cl   1.855340   2.416803   2.416820   2.416784   5.110873
                    6
     6  Cl   0.000000
 This structure is nearly, but not quite of a higher symmetry.
 Consider Symm=Loose if the higher symmetry is desired.
 Stoichiometry    CH3BrCl(1-)
 Framework group  C1[X(CH3BrCl)]
 Deg. of freedom    12
 Full point group                 C1      NOp   1
 Largest Abelian subgroup         C1      NOp   1
 Largest concise Abelian subgroup C1      NOp   1
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -1.357531    0.018931    0.160587
      2          1           0       -0.934127   -0.532296    0.990449
      3          1           0       -1.142606   -0.468949   -0.781523
      4          1           0       -1.010919    1.044423    0.156457
      5         35           0        1.875166   -0.026033   -0.221704
      6         17           0       -3.199880    0.044376    0.378279
 ---------------------------------------------------------------------
 Rotational constants (GHZ):         157.3227567           0.7546936           0.7546936
 Standard basis: 6-311++G(d,p) (5D, 7F)
 There are   126 symmetry adapted cartesian basis functions of A   symmetry.
 There are   121 symmetry adapted basis functions of A   symmetry.
   121 basis functions,   210 primitive gaussians,   126 cartesian basis functions
    31 alpha electrons       31 beta electrons
       nuclear repulsion energy       163.6052214303 Hartrees.
 NAtoms=    6 NActive=    6 NUniq=    6 SFac= 1.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=   121 RedAO= T EigKep=  2.79D-04  NBF=   121
 NBsUse=   121 1.00D-06 EigRej= -1.00D+00 NBFU=   121
 ExpMin= 3.50D-02 ExpMax= 4.40D+05 ExpMxC= 1.51D+04 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 Harris functional with IExCor=  402 and IRadAn=       5 diagonalized for initial guess.
 HarFok:  IExCor=  402 AccDes= 0.00D+00 IRadAn=         5 IDoV= 1 UseB2=F ITyADJ=14
 ICtDFT=  3500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         wScrn=  0.000000 ICntrl=       500 IOpCl=  0 I1Cent=   200000004 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.
 Keep R1 ints in memory in canonical form, NReq=29147525.
 Requested convergence on RMS density matrix=1.00D-08 within 128 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 SCF Done:  E(RB3LYP) =  -3074.40310419     A.U. after   11 cycles
            NFock= 11  Conv=0.92D-08     -V/T= 2.0014

 **********************************************************************

            Population analysis using the SCF Density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A)
       Virtual   (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A)
 The electronic state is 1-A.
 Alpha  occ. eigenvalues -- -482.55410-101.41536 -62.20707 -56.02488 -56.02480
 Alpha  occ. eigenvalues --  -56.02480 -10.08808  -9.32956  -8.41573  -7.09338
 Alpha  occ. eigenvalues --   -7.08474  -7.08474  -6.23879  -6.23845  -6.23845
 Alpha  occ. eigenvalues --   -2.35116  -2.35105  -2.35105  -2.35078  -2.35078
 Alpha  occ. eigenvalues --   -0.71479  -0.54683  -0.48719  -0.31141  -0.31141
 Alpha  occ. eigenvalues --   -0.27745  -0.18013  -0.18012  -0.05277  -0.05123
 Alpha  occ. eigenvalues --   -0.05123
 Alpha virt. eigenvalues --    0.10750   0.11988   0.14665   0.14665   0.17432
 Alpha virt. eigenvalues --    0.18065   0.19467   0.19467   0.20589   0.20590
 Alpha virt. eigenvalues --    0.21572   0.22245   0.22246   0.24720   0.27939
 Alpha virt. eigenvalues --    0.30397   0.30397   0.36215   0.40214   0.53170
 Alpha virt. eigenvalues --    0.53171   0.55019   0.58311   0.58312   0.61226
 Alpha virt. eigenvalues --    0.62058   0.62058   0.63991   0.76141   0.76141
 Alpha virt. eigenvalues --    0.79436   0.84245   0.88021   0.88021   0.89223
 Alpha virt. eigenvalues --    0.89224   0.94588   0.96185   0.96186   1.00579
 Alpha virt. eigenvalues --    1.00579   1.02228   1.05489   1.25731   1.25732
 Alpha virt. eigenvalues --    1.26341   1.56582   1.67177   1.67177   1.84390
 Alpha virt. eigenvalues --    1.84391   1.86045   2.10345   2.21692   2.21693
 Alpha virt. eigenvalues --    2.26749   2.26749   2.27395   2.45204   2.45205
 Alpha virt. eigenvalues --    2.55013   2.58700   2.63647   2.63649   2.66311
 Alpha virt. eigenvalues --    2.66312   3.08016   3.28716   3.86627   3.86630
 Alpha virt. eigenvalues --    4.39739   4.39742   4.40346   4.40347   4.48979
 Alpha virt. eigenvalues --    6.75569   7.80397   7.80398   7.84943   9.83601
 Alpha virt. eigenvalues --   24.02525  25.98193  25.98193  26.48254  48.07036
 Alpha virt. eigenvalues --  215.82160 290.01214 290.01214 290.054651020.82576
          Condensed to atoms (all electrons):
               1          2          3          4          5          6
     1  C    5.602027   0.400422   0.400418   0.400420  -0.045839  -0.108923
     2  H    0.400422   0.517827  -0.037764  -0.037760   0.003276  -0.038624
     3  H    0.400418  -0.037764   0.517798  -0.037759   0.003274  -0.038607
     4  H    0.400420  -0.037760  -0.037759   0.517843   0.003275  -0.038634
     5  Br  -0.045839   0.003276   0.003274   0.003275  35.921965   0.045953
     6  Cl  -0.108923  -0.038624  -0.038607  -0.038634   0.045953  17.176283
 Mulliken charges:
               1
     1  C   -0.648525
     2  H    0.192623
     3  H    0.192640
     4  H    0.192614
     5  Br  -0.931905
     6  Cl   0.002552
 Sum of Mulliken charges =  -1.00000
 Mulliken charges with hydrogens summed into heavy atoms:
               1
     1  C   -0.070647
     5  Br  -0.931905
     6  Cl   0.002552
 Electronic spatial extent (au):  <R**2>=           1262.2107
 Charge=             -1.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=             -4.8415    Y=              0.0674    Z=              0.5726  Tot=              4.8757
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -76.2154   YY=            -44.4920   ZZ=            -44.9295
   XY=              0.4401   XZ=              3.7510   YZ=             -0.0520
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -21.0031   YY=             10.7203   ZZ=             10.2828
   XY=              0.4401   XZ=              3.7510   YZ=             -0.0520
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=             43.8741  YYY=              0.5049  ZZZ=             -3.1458  XYY=              8.5684
  XXY=             -0.3828  XXZ=             -3.1708  XZZ=              8.7757  YZZ=             -0.9711
  YYZ=             -0.9204  XYZ=             -0.0692
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=          -1672.7451 YYYY=            -76.8960 ZZZZ=            -96.8715 XXXY=             12.1855
 XXXZ=            103.5393 YYYX=              9.1896 YYYZ=             -1.0826 ZZZX=             85.5020
 ZZZY=             -1.4701 XXYY=           -265.8117 XXZZ=           -271.1797 YYZZ=            -28.9601
 XXYZ=             -0.4638 YYXZ=             28.3094 ZZXY=              4.1388
 N-N= 1.636052214303D+02 E-N=-7.680468815879D+03  KE= 3.070011960571D+03
 Unable to Open any file for archive entry.
 1\1\GINC-NODE376\SP\RB3LYP\6-311++G(d,p)\C1H3Br1Cl1(1-)\ROOT\11-Nov-20
 20\0\\# b3lyp/6-311++g(d,p) output=wfn\\Single Point Energy\\-1,1\C,0,
 -1.497485,-0.000103,0.00009\H,0,-1.16687,-0.874824,-0.545208\H,0,-1.16
 6746,0.90947,-0.48476\H,0,-1.166954,-0.035017,1.030294\Br,0,1.758048,0
 .000017,0.\Cl,0,-3.352825,0.000007,-0.00004\\Version=ES64L-G16RevC.01\
 State=1-A\HF=-3074.4031042\RMSD=9.193e-09\Dipole=-1.9182594,-0.0000852
 ,0.0000622\Quadrupole=-15.9495102,7.9747186,7.9747916,0.0000248,-0.000
 2636,-0.0000065\PG=C01 [X(C1H3Br1Cl1)]\\@
 The archive entry for this job was punched.

 Writing a WFN file to "frame.wfn"


 WHAT, THEN, IS TIME?  IF NO ONE ASKS ME, I KNOW WHAT IT IS.
 IF I WISH TO EXPLAIN WHAT IT IS TO HIM WHO ASKS ME, I DO NOT KNOW.
                                   -- ST. AUGUSTINE (FIFTH CENTURY)
 Job cpu time:       0 days  0 hours  0 minutes 27.3 seconds.
 Elapsed time:       0 days  0 hours  0 minutes 14.2 seconds.
 File lengths (MBytes):  RWF=     10 Int=      0 D2E=      0 Chk=      2 Scr=      1
 Normal termination of Gaussian 16 at Wed Nov 11 19:03:01 2020.
